data/human/human missions.txt

```diff
@@ -6890,3 +6890,1716 @@ mission "Hogshead's Dilemma II"
+			choice
+				`	"Fine. I'll drop you off at the next port."`
+					goto port
+				`	"Is there somewhere you'd like to go?"`
+					goto somewhere
+				`	"I know where you're going. The airlock."`
+
+			action
+				fail
+			`	The feeble man screams as you drag him out of the crate and toward the airlock. He ineptly squirms as you shove him pass the bulkhead seals. He falls to his knees, shudders with terror, and his face leaks tears. You close the airlock, silencing his shrill screams, and hit the button to open the outside airlock. The man is promptly sucked out into cold, cruel space and crystallizes before becoming this system's newest comet.`
+
+			label port
+			action
+				fail
+				set "timothy radrickson: port"
+			`	Tears squeeze out the man's sad eyes. "Thank you", he whispers, "thank you."`
+			`	You get the man cleaned up and settled, and head then go back to running your ship.`
+
+			label somewhere
+			`	Timothy looks at you, his eyes wide and full of tears. "Rand", he whispers. "Rand. I can make good money there. I can turn my life around. B-b-but I'd be okay with anywhere. You d-d-don't have to.."`
+			`	You cut him off. "It's fine, Timothy. I'll take you to Rand."`
+			`	Timothy breaks down in tears. "Thank you. Thank you. Thank you..." he mumbles over and over again.`
+			`	You help him out of the crate and get him cleaned up.`
+				accept
+
+	on complete
+		event "timothy radrickson: on rand" 340
+		conversation
+			`During the trip Timothy's spirits and health improved, and by the time you arrive on Rand he seems like a different man. Before he steps out of the airlock he turns to you one more time. "Thank you so much, Captain <last>." He shakes your hand."`
+			`	"I mean it, from the bottom of my heart. You've saved my life, and given me hope. I won't forget this; or you. I could never thank you enough."`
+			`	His eyes glistening, Timothy turns from you and exits your ship. Hopefully he finds what he is looking for on Rand.`
+
+
+mission "Timothy Radrickson 1a: Accepted"
+	landing
+	name "Stowaway to <planet>"
+	description "You have found Timothy on your ship stowed away in a foodstuff crate, on the run from somebody. He requests you to bring him to <destination>, where he can find work."
+	source
+		government "Republic" "Free Worlds" "Independent" "Pirate" "Neutral"
+	destination "Rand"
+	to offer
+		has "Timothy Radrickson 1: Stowaway: active"
+	on abort
+		fail "Timothy Radrickson 1: Stowaway"
+
+
+event "timothy radrickson: on rand"
+event "timothy radrickson: mummy"
+
+
+mission "Timothy Radrickson 2: mummy"
+	landing
+	invisible
+	to offer
+		has "Timothy Radrickson 1: Stowaway: failed"
+		has "event: timothy radrickson: mummy"
+	on enter
+
+	on enter
```

Comment:
Still duplicated here.

Reply:
That's on purpose. Duplicated on enter means that it should happen when you are in space after a jump, and not on planet. 
So you notice your ship stinking of dead-dude in space and not on world.

Reply:
Okay, you've learned me something.